src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/gateways/BankHolidayApiGateway.kt

```diff
@@ -1,17 +1,14 @@
 package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.gateways
 
 import org.springframework.beans.factory.annotation.Qualifier
-import org.springframework.cache.annotation.Cacheable
 import org.springframework.core.ParameterizedTypeReference
 import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
-import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.config.CacheConfiguration
 
 @Service
 class BankHolidayApiGateway(@Qualifier("bankHolidayApiWebClient") private val webClient: WebClient) {
   private inline fun <reified T> typeReference() = object : ParameterizedTypeReference<T>() {}
 
-  @Cacheable(CacheConfiguration.BANK_HOLIDAYS_CACHE_NAME)
```

Comment:
If we are doing this then we have to change the health endpoint.

I think the best approach for the health check is to (somehow) just check the cache has been updated in the last - say- 4 weeks.